### DISCUSSION TITLE
requeue is confusing

### DISCUSSION THREAD

**Discussion by xxchan** [Mergifyio/mergify#5044] [General]
Our config is:
- rules for going into merge queue: test pass
- rules for merging: test pass

https://github.com/risingwavelabs/risingwave/blob/c3ba77c7f7d647676269d008a2ab17fb1598075b/.mergify.yml

When CI fails due to flaky test, `requeue` command will not work. Should update branch/push empty commit instead. 

Did we use it wrongly?

**Comment by jd** [comment]
Hi @xxchan,

IIUC `requeue` won't work since your PR is not in a state where it can be queued, and if it's already up to date, there's no way for Mergify to update the PR and re-trigger the CI. In that case you need to retrigger the flaky test CI yourself to validate again the rules condition to be queued.

**Reply by xxchan** [reply]
Yes, this is the same as my current understanding. But it was confusing at the beginning. What about adding some notes about requeue in the doc?
